src/activities/drawnumber/drawnumber.js

```diff
@@ -20,14 +20,17 @@
  *   along with this program; if not, see <http://www.gnu.org/licenses/>.
  */
 .pragma library
-.import QtQuick 2.0 as Quick
+.import QtQuick 2.2 as Quick
+.import GCompris 1.0 as GCompris //for ApplicationInfo
+.import "qrc:/gcompris/src/core/core.js" as Core
 
 var currentLevel = 0
 var items
-var clickanddrawflag
-
+var mode
 var pointPositions = []
 var linePropertiesArray = []
+//array to play sound of different alphabets in different levels
+var soundplay = ["d","c","l","a","o","p","m","n","r","w","s","z","u"]
```

Comment:
can you incorporate it in the dataset instead of having a separate variable?
This way, if we want to add sounds in another activity, we just have to update the dataset, not this js file